client/src/components/Button/index.js

```diff
@@ -1,7 +1,11 @@
 import React from 'react';
 
-function Button({ textToDisplay }) {
-  return <button className="button">{textToDisplay}</button>;
+function Button({ textToDisplay, classes, callback }) {
+  return (
+    <button className={classes} onClick={callback}>
+      {textToDisplay}
+    </button>
+  );
 }
```

Comment:
don't you think it's way more complicated to use buttons like this?

Reply:
Could you suggest any specific solution to this problem?

Reply:
@synowa just use buttons as standard jsx element. There's no need to extract it to a separate component right now.
Let's say:
```jsx
<codeBefore/>
<button anyPropsINeedRightNow>anyText</button>
<codeAfter/>
```
Pros:
1. No imports (less messy components directory in the future)
2. Every additional prop that's not defined in `Button` component would have to be declared

If your not able to define how many props you'll then you should use the spread operator
```jsx
const Example = (props) => (
    <button ...props>{props.children}</button>
);
```

_props.children stands for any child you put inside this component, eg. text or another component_

Reply:
I actually had in mind a very specific button, with specific style and function - a go back button, using props.history.goBack(). But I didn't have time to implement it yet. So far yes, it is reduntant.